File: javascript/iniciante/1007.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
});

const values = [];

rl.on("line", (line) => {
  values.push(...line.trim().split(/\s+/).filter(Boolean));

  if (values.length >= 4) {
    const A = Number(values[0]);
    const B = Number(values[1]);
    const C = Number(values[2]);
    const D = Number(values[3]);
    const difference = A * B - C * D;
    console.log(`DIFERENCA = ${String(difference)}`);
    rl.close();
  }
});

File: swift/iniciante/1007.swift
import Foundation

if let inputA = readLine(), let inputB = readLine(), let inputC = readLine(),
   let inputD = readLine(),
   let A = Int(inputA),
   let B = Int(inputB),
   let C = Int(inputC),
   let D = Int(inputD)
{
    let difference = (A * B) - (C * D)

    print("DIFERENCA = \(difference)")
}
